committing.rst

```diff
@@ -218,14 +218,19 @@ the role is converted into a hyperlink. Example news entry with Sphinx::
 
 The inline Sphinx roles can be used to assist readers in finding more
 information and context on the changes made. If the inline link does not
-provide additional context, an inline reST code block should be used instead::
+provide additional context, an inline reST code block can be used instead::
 
    ``<object>``
 
 Before using any Sphinx roles, ensure that a corresponding entry exists
-within the documentation. Also, `build the HTML locally using make 
+within the documentation. When editing multiple news entries at once in
+"What's New" or Misc/NEWS.d/, `build the HTML locally using make 
 <https://devguide.python.org/documenting/#using-make-make-bat>`_ to
-verify that the link leads to the correct location.
+verify that the links lead to the correct locations. This does not need to be
+done for individual PR news entries.
```

Comment:
I included this because of what you said earlier:

> Run "make html" locally is huge pain for reviewer. I check out the PR only when it's really needed. I review the code only on Github in 80% PRs.

Reply:
> I included this because of what you wrote earlier:
> 
> > Run "make html" locally is huge pain for reviewer. I check out the PR only when it's really needed. I review the code only on Github in 80% PRs.

That's very bad reason to allow no-check merging.  That can be a reason to just using inline  code block.

> I can remove this part entirely though if that would be preferred.

Yes, please remove it.  When using rich formatting, a careful review is required.  It is same to normal document changes.